### PR TITLE
openbsd adding kinfo_vmentry

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: src/unix/bsd/netbsdlike/openbsd/mod.rs [hunk 1]
```diff
@@ -384,6 +384,23 @@ s! {
...
+        pub kve_fspace: ::c_ulong,
+        pub kve_fspace_augment: ::c_ulong,
+        pub kve_offset: u64,
+        pub kve_wired_count: ::c_int,
+        pub kve_etype: ::c_int,
+        pub kve_protection: ::c_int,
+        pub kve_max_protection: ::c_int,
+        pub kve_advice: ::c_int,
+        pub kve_inheritance: ::c_int,
+        kve_flags: u8,
```
Comment (semarie) [inline src/unix/bsd/netbsdlike/openbsd/mod.rs:402]: maybe adding `pub` to `kve_flags` too ?

**Review by semarie — CHANGES_REQUESTED** [review]
just tested (with `kve_flags` changed) and it is fine.

**Review by semarie — APPROVED** [review]
fine with me

**Review by JohnTitor — APPROVED** [review]

**Pull request merged**